### PR TITLE
Rename views.py module to view.py

### PR DESCRIPTION
**Pull request by emlaver**
## What:

Rename views.py to view.py to match naming pattern for modules within this library.
#132 

## How:

- Rename views.py to view.py
- Update all import references in code and tests
- Update Sphinx documentation .rst files
- Update docsting comments

## Testing:

- No additional tests
- All tests using views module now use view module

## Reviewers:

reviewer @alfinkel 
reviewer @ricellis

### CODE REVIEW

File: tests/unit/mocked/view_test.py [hunk 1]
```diff
@@ -13,21 +13,21 @@
...
 
-from cloudant.views import Code, View
+from cloudant.view import Code, View
 from cloudant.design_document import DesignDocument
 from cloudant.result import Result
 
 
 class CodeTests(unittest.TestCase):
-    """tests for cloudant.views.Code, not much to test here yet..."""
+    """tests for cloudant.view.Code, not much to test here yet..."""
```
Comment (ricellis) [inline tests/unit/mocked/view_test.py]: Can probably update this comment, looks like there are some tests.

**Comment by alfinkel** [comment]
+1

File: tests/unit/mocked/view_test.py (hunk 1 above)
Comment (alfinkel) [inline tests/unit/mocked/view_test.py]: I think the comment is correct.  There's just the one test.

**Comment by ricellis** [comment]
I couldn't find any other references to the `views` module so +1

File: tests/unit/mocked/view_test.py (hunk 1 above)
Comment (alfinkel) [inline tests/unit/mocked/view_test.py]: Looking at the code, the one "test" does not even assert anything so there is actually < 1 tests :).

In any event, these mocked tests will soon be removed anyway.

Comment (emlaver) [inline tests/unit/mocked/view_test.py]: @alfinkel Yes, I agree, this class will be removed in #113.

**Pull request merged**